src/vs/base/browser/ui/actionbar/actionbar.ts

```diff
@@ -496,6 +496,28 @@ export class ActionBar implements IActionRunner {
 			this.actionsList.setAttribute('aria-label', this.options.ariaLabel);
 		}
 
+		if (this.options.isMenu) {
```

Comment:
This is pretty rough... At some point it would just be better for the menu to implement menu items on its own. Could we get this behaviour in some other way (eg `mouseoverHandler`)?

Reply:
Just the knowledge of menus, from the action bar part. It's also not urgent or important, just to keep in mind not to push that much further.